### ISSUE TITLE
508-defect-3 [COGNITION]: Text that is visually styled and intended as headings SHOULD be headings

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#19323]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** _Josh_

## User Story or Problem Statement

As a screen reader user, I want to be able to navigate by headings on the page.

## Details

The heading "What if I need help filling out my application?" is not accessible by screen reader users as it is written as a `<strong>` instead of a `<hx>`. This causes several issues:
- It will not appear in a screen reader user's natural heading order, which makes it less accessible
- It may confuse low vision screen reader users who expect to be able to navigate to it using the `h` key but cannot
- A lack of visual consistency will fail to meet future [WCAG 3.0 criteria regarding visually distinct headings](https://www.w3.org/WAI/GL/WCAG3/2020/methods/visually-distinct/)

## Acceptance Criteria

- [x] "What if I need help filling..." should be written as a heading and share the same heading level as the heading "to fill out this application..."

### Before
Currently it is wrapped in `strong` within a `p`

```html
<p>
  <strong>
    What if I need help filling out my application?
  </strong> 
  An accredited representative, with a Veterans Service Organization (VSO), can help you fill out your claim. <a href="/disability-benefits/apply/help/index.html">Get help filing your claim</a>
</p>
```

### After
Pull it out of the `p` and write it as an `h`

```html
<h?>
    What if I need help filling out my application?
</h?>
<p>
  An accredited representative, with a Veterans Service Organization (VSO), can help you fill out your claim. <a href="/disability-benefits/apply/help/index.html">Get help filing your claim</a>
</p>
```

## WCAG or Vendor Guidance (optional)

* [WCAG: Visually distinct headings](https://www.w3.org/WAI/GL/WCAG3/2020/methods/visually-distinct/)

## Screenshots or Trace Logs

<img width="1333" alt="Screen Shot 2021-02-01 at 11 14 04 AM" src="https://user-images.githubusercontent.com/14154792/106487216-749d7600-6480-11eb-84ae-499a48afe8a6.png">
<img width="1792" alt="Screen Shot 2021-02-01 at 11 14 50 AM" src="https://user-images.githubusercontent.com/14154792/106487217-749d7600-6480-11eb-8875-050b2613a30f.png">
<img width="1593" alt="Screen Shot 2021-02-01 at 11 28 02 AM" src="https://user-images.githubusercontent.com/14154792/106487455-ab738c00-6480-11eb-9572-57ffc7e4501f.png">
